apps/designer/app/designer/features/topbar/sync-status.tsx

```diff
@@ -1,45 +1,48 @@
-import { Flex, keyframes, AccessibleIcon } from "~/shared/design-system";
+import {
+  Flex,
+  keyframes,
+  AccessibleIcon,
+  styled,
+  Box,
+} from "~/shared/design-system";
 import { CheckIcon, DotsHorizontalIcon } from "~/shared/icons";
 import { useSyncStatus } from "../../shared/nano-states";
 
 const iconSize = 15;
 
+const StyledCheckIcon = styled(CheckIcon, {
```

Comment:
cc @thysultan instead of nested selector

Reply:
alternative is to make all icons accept a css prop so we don't have to use styled